### PR TITLE
[Rules] Restrict Finishing Blow to only Fleeing NPC's.

### PR DESCRIPTION
**Pull request by fryguy503**
Default is false to maintain current design

### CODE REVIEW

File: zone/attack.cpp [hunk 1]
```diff
@@ -5219,9 +5219,16 @@ bool Mob::TryFinishingBlow(Mob *defender, int64 &damage)
...
 
-		if (FB_Level && FB_Dmg && (defender->GetLevel() <= FB_Level) &&
-			(ProcChance >= zone->random.Int(1, 1000))) {
-
+		if (
+			(
+				(RuleB(Combat, FinishingBlowOnlyWhenFleeing) && !defender->currently_fleeing) ||
+				!RuleB(Combat, FinishingBlowOnlyWhenFleeing)
+			) &&
+			FB_Level &&
```
Comment (Kinglykrab) [inline zone/attack.cpp:5227]: `finishing_blow_level`

File: zone/attack.cpp [hunk 2]
```diff
@@ -5219,9 +5219,16 @@ bool Mob::TryFinishingBlow(Mob *defender, int64 &damage)
...
-		if (FB_Level && FB_Dmg && (defender->GetLevel() <= FB_Level) &&
-			(ProcChance >= zone->random.Int(1, 1000))) {
-
+		if (
+			(
+				(RuleB(Combat, FinishingBlowOnlyWhenFleeing) && !defender->currently_fleeing) ||
+				!RuleB(Combat, FinishingBlowOnlyWhenFleeing)
+			) &&
+			FB_Level && 
+			FB_Dmg &&
```
Comment (Kinglykrab) [inline zone/attack.cpp:5228]: `finishing_blow_damage`

File: zone/attack.cpp [hunk 3]
```diff
@@ -5219,9 +5219,16 @@ bool Mob::TryFinishingBlow(Mob *defender, int64 &damage)
...
-
+		if (
+			(
+				(RuleB(Combat, FinishingBlowOnlyWhenFleeing) && !defender->currently_fleeing) ||
+				!RuleB(Combat, FinishingBlowOnlyWhenFleeing)
+			) &&
+			FB_Level && 
+			FB_Dmg && 
+			defender->GetLevel() <= FB_Level &&
+			ProcChance >= zone->random.Int(1, 1000)
```
Comment (Kinglykrab) [inline zone/attack.cpp:5230]: `proc_chance`

File: zone/attack.cpp [hunk 4]
```diff
@@ -5219,9 +5219,16 @@ bool Mob::TryFinishingBlow(Mob *defender, int64 &damage)
...
-			(ProcChance >= zone->random.Int(1, 1000))) {
-
+		if (
+			(
+				(RuleB(Combat, FinishingBlowOnlyWhenFleeing) && !defender->currently_fleeing) ||
+				!RuleB(Combat, FinishingBlowOnlyWhenFleeing)
+			) &&
+			FB_Level && 
+			FB_Dmg && 
+			defender->GetLevel() <= FB_Level &&
```
Comment (Kinglykrab) [inline zone/attack.cpp:5229]: `finishing_blow_level`

**Review by Akkadius — APPROVED** [review]

**Pull request merged**